h/models/feature.py

```diff
@@ -21,6 +21,7 @@
     'orphans_tab': "Show the orphans tab to separate anchored and unanchored annotations?",
     'search_for_doi': "Use DOI metadata when searching for annotations on the current page?",
     'total_shared_annotations': "Show the total number of shared annotations for users and groups?",
+    'overlay_highlighter': "Use the new overlay highlighter?"
```

Comment:
On behalf of a pedant who shall remain nameless (😉) I'll note that this list should be sorted alphabetically.